server/routerlicious/packages/services-core/src/cache.ts

```diff
@@ -22,4 +22,14 @@ export interface ICache {
      */
      delete?(key: string): Promise<boolean>;
 
+     /**
+     * Increments key value by 1. If the key does not exist, its value will be first set to 0 and then incremented.
+     */
+    incr(key: string): Promise<number>;
+
+    /**
+     * Decrements key value by 1. If the key does not exist, its value will be first set to 0 and then decremented.
+     */
+    decr(key: string): Promise<number>;
```

Comment:
I think this will leave everyone who implements `ICache` fail their build. Should we do `incr?` and `decr`?